client/my-sites/domains/style.scss

```diff
@@ -19,22 +19,6 @@
 		margin-bottom: auto;
 	}
 
-	.popover-cart {
```

Comment:
We can also clean this up from https://github.com/Automattic/wp-calypso/blob/8aa46e6a0a8517cfc087c4884316ed76d93ce34e/client/my-sites/domains/domain-management/components/breadcrumbs/style.scss#L141